modules/tinymce/src/plugins/table/main/ts/api/QueryCommands.ts

```diff
@@ -22,9 +23,9 @@ const registerQueryCommands = (editor: Editor, actions: TableActions, selections
   Obj.each({
     mceTableRowType: () => actions.getTableRowType(editor),
     mceTableCellType: () => actions.getTableCellType(editor),
-    mceTableColType: () => TableSelection.getSelectionStartCell(editor).bind((cell) =>
+    mceTableColType: () => TableSelection.getSelectionStartCell(Util.getSelectionStart(editor)).bind((cell) =>
       getTableFromCell(cell).map((table): string => {
-        const targets = TableTargets.forMenu(selections, table, cell);
+        const targets = TableTargets.forMenu(selections, table, cell, Ephemera);
```

Comment:
I think Lee mentioned this elsewhere but why pass it to a function if its just a constant that could be imported where it's used

Reply:
Yep, I'm going to remove this.